daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -901,10 +901,12 @@ func (devices *DeviceSet) loadMetadata(hash string) *devInfo {
 
 	jsonData, err := ioutil.ReadFile(devices.metadataFile(info))
 	if err != nil {
+		logrus.Errorf("devmapper: Failed to read %s with err: %v", devices.metadataFile(info), err)
```

Comment:
@cpuguy83 , thanks very much for taking time to review this PR. I have amended this PR according to your suggestion, please help to review it again, thanks.